packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -346,6 +347,13 @@ export default {
 <style lang="scss" module>
 @use "@justeat/fozzie/src/scss/fozzie" as f;
 
+.c-selfExclusion {
+    display: flex;
+    justify-content: center;
+    width: 100%;
```

Comment:
divs are 100% width by default. Is this really needed?

Reply:
Divs are display: block but aren't 100% of width. This one is changed to display: flex and width 100% is necessary.